src/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarCommercialButtons.tsx

```diff
@@ -697,7 +691,11 @@ const ARTWORK_FRAGMENT = graphql`
       }
     }
     collectorSignals {
-      primaryLabel(ignore: [PARTNER_OFFER])
+      partnerOffer {
+        endAt
+      }
+      increasedInterest
+      curatorsPick
```

Comment:
Now on the MP schema side, we separate out the fields requiring auth (e.g. `partnerOffer`) from fields without requiring auth (e.g. `primaryLabel`), we'll need to request them both.